vtr_flow/arch/multi_die/stratixiv_3d/3d_full_OPIN_inter_die_stratixiv_arch.timing.xml

```diff
@@ -5115,9 +5115,11 @@
               while keeping the switch delay a reasonable (see comment in <segmentlist>) portion of the overall delay.
        -->
     <switch type="mux" name="seg4_driver" R="450" Cin="0.60e-15" Cout="4.82e-15" Tdel="59e-12" mux_trans_size="2.630740" buf_size="27.645901"/>
+    <switch type="mux" name="seg4_inter_layer_driver" R="0.0" Cin="0.0" Cout="0.0" Tdel="138.82e-12" mux_trans_size="2.630740" buf_size="27.645901"/>
```

Comment:
Could comment why this delay is what it is (reasoning behind taking it from Koios).